cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -404,8 +402,9 @@ def set_args(self, request, verbose=False):
 
         self.par_dic = args.to_dict()
 
-        # if passed, disregard it, since it is nowhere necessary within the dispatcher-app
-        self.par_dic.pop('use_scws', None)
+        # if passed, disregard it, since it is nowhere necessary within the dispatcher-app,
+        # but re-attach it to the url within the email when sending it
+        self.use_scws = self.par_dic.pop('use_scws', None)
```

Comment:
maybe entire logic of determining if we `use_scws` can be kept in a separate method, called before this pop.
It would use combination of frontend-way and api-way to derive use_scws.

Then, product URLs are always made for frontend and should include `use_scws`.

API codes do not.